packages/venia-ui/lib/components/WishlistPage/wishlistItem.js

```diff
@@ -67,65 +59,66 @@ const WishlistItem = props => {
 
     const classes = mergeClasses(defaultClasses, props.classes);
 
-    const optionElements = configurableOptions.map(option => {
-        const {
-            id,
-            option_label: optionLabel,
-            value_label: valueLabel
-        } = option;
+    const optionElements = useMemo(() => {
+        return configurableOptions.map(option => {
+            const {
+                id,
+                option_label: optionLabel,
+                value_label: valueLabel
+            } = option;
+
+            const optionString = `${optionLabel} : ${valueLabel}`;
 
-        const optionString = `${optionLabel} : ${valueLabel}`;
+            return (
+                <span className={classes.option} key={id}>
+                    {optionString}
+                </span>
+            );
+        });
+    }, [classes.option, configurableOptions]);
 
-        return (
-            <span className={classes.option} key={id}>
-                {optionString}
-            </span>
-        );
-    });
+    const imageProps = {
+        classes: {
+            image:
+                stockStatus === 'OUT_OF_STOCK'
+                    ? classes.image_disabled
+                    : classes.image
+        },
+        ...talonProps.imageProps
+    };
 
     return (
         <div className={classes.root}>
-            <Image
-                alt={imageLabel}
-                classes={{ image: classes.image }}
-                src={url}
-                width={400}
-            />
+            <Image {...imageProps} />
```

Comment:
For some reason the first image in my list is lower resolution and blurry. I can give you the account credentials to check against. The same account on `develop` does not have this issue.

[![Image from Gyazo](https://i.gyazo.com/4aeb771e1244c8191fa953a6ab9a95a6.png)](https://gyazo.com/4aeb771e1244c8191fa953a6ab9a95a6)

Reply:
I've seen this once before, and it was from switching between mobile and desktop. If the browser caches a lower res image when in mobile, it'll re-use it when you switch to desktop. Doesn't seem like a realistic use case, but I'll see what the options are to remedy.

Reply:
Yea I didn't know how to repro so if it doesn't occur out of "normal" use I wouldn't worry about it.